packages/components/src/DataList/DataList.stories.tsx

```diff
@@ -12,43 +21,113 @@ export default {
     viewMode: "story",
   },
   // Comment this out to make it show up in storybook
-  excludeStories: ["Basic"],
+  // excludeStories: ["Basic"],
```

Comment:
Note to self: Don't forget to comment out
```suggestion
  excludeStories: ["Basic"],
```